src/core/tests/CMakeLists.txt

```diff
@@ -169,6 +169,7 @@ set(SRC
     type_prop/if.cpp
     type_prop/interpolate.cpp
     type_prop/irdft.cpp
+    type_prop/is_nan.cpp
```

Comment:
There are common attribute visitor tests for unary operations without attributes, such test instance also should be added for `IsNaN` op.
Example for for "Negative" op:
https://github.com/openvinotoolkit/openvino/blob/78d6f68de4f3b4d358b8e8fc0b066758e500f9cc/src/core/tests/visitors/op/negative.cpp#L7-L10

Reply:
done